### PR TITLE
1.4 doc updates

### PR DESCRIPTION
**Pull request by idlewis**
RCO portion of https://github.com/OpenLiberty/open-liberty-operator/issues/631

### CODE REVIEW

File: doc/user-guide-v1.adoc [hunk 1]
```diff
@@ -133,6 +133,9 @@ Each `RuntimeComponent` CR must at least specify the `.spec.applicationImage` fi
 | `deployment.annotations`   | Annotations to be added only to the deployment and resources owned by the deployment.
 | `deployment.updateStrategy`   | A field to specify the update strategy of the deployment. For examples, see link:++https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy++[updateStrategy]
 | `deployment.updateStrategy.type`   | The type of update strategy of the deployment. The type can be set to `RollingUpdate` or `Recreate`, where `RollingUpdate` is the default update strategy.
+| `dns` | DNS settings for the application pods. For more information, see link:#++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#configure-dns-specdnspolicy-and-specdnsconfig[Configure DNS]
+| `dns.config` | The DNS Config for the application pods.
+| `dns.spec` | The DNS Policy for the application pod. Defaults to ClusterFirst.
```
Comment (leochr) [inline doc/user-guide-v1.adoc:138]: `dns.spec` should be `dns.policy`

File: doc/user-guide-v1.adoc [hunk 2]
```diff
@@ -133,6 +133,9 @@ Each `RuntimeComponent` CR must at least specify the `.spec.applicationImage` fi
 | `deployment.annotations`   | Annotations to be added only to the deployment and resources owned by the deployment.
 | `deployment.updateStrategy`   | A field to specify the update strategy of the deployment. For examples, see link:++https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy++[updateStrategy]
 | `deployment.updateStrategy.type`   | The type of update strategy of the deployment. The type can be set to `RollingUpdate` or `Recreate`, where `RollingUpdate` is the default update strategy.
+| `dns` | DNS settings for the application pods. For more information, see link:#++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#configure-dns-specdnspolicy-and-specdnsconfig[Configure DNS]
```
Comment (leochr) [inline doc/user-guide-v1.adoc:136]: `Configure DNS` doesn't render as a link

File: doc/user-guide-v1.adoc [hunk 3]
```diff
@@ -203,6 +206,7 @@ Each `RuntimeComponent` CR must at least specify the `.spec.applicationImage` fi
 | `statefulSet.storage.volumeClaimTemplate` | A YAML object representing a link:++https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components++[volumeClaimTemplate] component of a `StatefulSet`.
 | `statefulSet.updateStrategy`   | A field to specify the update strategy of the StatefulSet. For examples, see link:++https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies++[updateStrategy]
 | `statefulSet.updateStrategy.type`   | The type of update strategy of the StatefulSet. The type can be set to `RollingUpdate` or `OnDelete`, where `RollingUpdate` is the default update strategy.
+| `tolerations` | Tolerations to be added to application pods. Tolerations allow the scheduler to schedule pods on nodes with matching taints. For more information, see link:#++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#configure-tolerations[Configure tolerations].
```
Comment (leochr) [inline doc/user-guide-v1.adoc:209]: `Configure tolerations ` doesn't render as a link

File: doc/user-guide-v1.adoc [hunk 4]
```diff
@@ -365,6 +369,8 @@ Browse the `RuntimeComponent` examples to learn how to use custom resource (CR)
   - link:++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#generating-certificates-with-certificate-manager++[Generating certificates with certificate manager]
   - link:++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#generating-certificates-with-red-hat-openshift-service-ca++[Generating certificates with Red Hat OpenShift service CA (`.spec.service.annotations`)]
   - link:++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#specifying-certificates-for-a-secret-route-and-service++[Specifying certificates for a secret Route and Service (`.spec.service.certificateSecretRef` and `.spec.route.certificateSecretRef`)] 
+* link:++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#configure-tolerations++[Configure tolerations (`.spec.tolerations`)]
+* link:++https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc#configure-dns-specdnspolicy-and-specdnsconfig++[Configure DNS (`.spec.dns.policy` and `.spec.dns.config`)]
```
Comment (leochr) [inline doc/user-guide-v1.adoc:373]: The end part of the link `#configure-tolerations` should be `#configure-tolerations-spectolerations` I think

**Review by leochr — COMMENTED** [review]
@idlewis Thanks for the PR. Added some comments

**Comment by idlewis** [comment]
I think I've addressed all of these, ready for re-review

**Review by leochr — APPROVED** [review]
@idlewis Thanks for the updates. Looks good.

**Pull request merged**